gradle.properties

```diff
@@ -1,7 +1,9 @@
-projectVersion=2.3.1-SNAPSHOT
+projectVersion=2.4.0-SNAPSHOT
 micronautDocsVersion=1.0.24
 micronautBuildVersion=1.1.5
-micronautVersion=2.3.1
+micronautDiscoveryClientVersion=2.0.1
```

Comment:
Is there a reason you downgraded the version from 2.2.4 to 2.0.1?

Reply:
I'll fix that.  Just a miss on the merge.

Reply:
Done.